static/styles/courses/course.scss

```diff
@@ -40,6 +40,10 @@
 		}
 	}
 
+	.dropdown-toggle:hover {
+        color: $primaryColor !important;
+    }
+
```

Comment:
Is this part of the story? If so please avoid using important in css. You should have a proper css structure instead